### PR TITLE
UCRM Client Signup Plugin

### PR DESCRIPTION
**Pull request by charuwts**
When installed onto UCRM the plugin public URL will display a form that anyone can enter valid info into to create a client.

### CODE REVIEW

**Comment by charuwts** [comment]
@janprochazkacz Vender file removed

File: plugins/UCRM-Client-Signup/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/data
+/vender
```
Comment (janprochazkacz) [inline plugins/UCRM-Client-Signup/.gitignore]: Typo. Change it to  ```/vendor/```

File: plugins/UCRM-Client-Signup/.gitignore [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
 # compiled output
-/data
-/vender
-/temp
+/data/
+/vender/
```
Comment (keksa) [inline plugins/UCRM-Client-Signup/.gitignore]: should be `vendor`, with `o`, not `e`

File: plugins/UCRM-Client-Signup/.gitignore [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/data/
+/vender/
```
Comment (keksa) [inline plugins/UCRM-Client-Signup/.gitignore]: should be `/vendor/` with `o` instead of `e`

File: plugins/UCRM-Client-Signup/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# UCRM Client Signup
+A [UCRM plugin](https://github.com/Ubiquiti-App/UCRM-plugins) that provides a frontend form for clients to signup.
+
+_Developed by [Charuwts, LLC](https://charuwts.com)_
+
+When installed onto UCRM the plugin public URL will display a form that anyone can enter valid info into to create a client.
+
+## Instructions
+
+_*Not ready for Production.*_
```
Comment (keksa) [inline plugins/UCRM-Client-Signup/README.md]: why not?

File: plugins/UCRM-Client-Signup/.gitignore (hunk 1 above)
Comment (charuwts) [reply plugins/UCRM-Client-Signup/.gitignore]: Thanks. @janprochazkacz  :+1:

File: plugins/UCRM-Client-Signup/.gitignore (hunk 2 above)
Comment (janprochazkacz) [inline plugins/UCRM-Client-Signup/.gitignore]: It is ``/vendor/`` not ``/vender/`` 😄

Comment (charuwts) [reply plugins/UCRM-Client-Signup/.gitignore]: :flushed: :grin:

File: plugins/UCRM-Client-Signup/README.md (hunk 4 above)
Comment (charuwts) [reply plugins/UCRM-Client-Signup/README.md]: Technically it is now, but I haven't written many tests so I personally wouldn't consider it secure/stable. But if someone wanted to use it live I wouldn't have a problem with it. Would you suggest I remove the comment?

**Comment by charuwts** [comment]
The javascript can be found here. https://github.com/charuwts/UCRM-Client-Signup-Form

File: plugins/UCRM-Client-Signup/README.md (hunk 4 above)
Comment (keksa) [reply plugins/UCRM-Client-Signup/README.md]: I will get back to you about it. :)

Comment (keksa) [reply plugins/UCRM-Client-Signup/README.md]: IMO it should stay, but maybe more verbose with something like you just commented here.

Comment (keksa) [reply plugins/UCRM-Client-Signup/README.md]: Other than that, I think this can be merged. @UBNT-Petr

Comment (charuwts) [reply plugins/UCRM-Client-Signup/README.md]: Thank you @keksa

Comment (charuwts) [reply plugins/UCRM-Client-Signup/README.md]: FYI, I added clarification to the readme. @keksa @UBNT-Petr

**Comment by charuwts** [comment]
Thank you @UBNT-Petr I'll look into applying a fix and removing the phone validation as soon as possible.

**Pull request merged**